app/src/Bomberman.hpp

```diff
@@ -20,7 +20,7 @@ static const char *WINDOW_TITLE = "Bomberman";
 class Bomberman : private AMainLoop
 {
 private:
-	sf::RenderWindow window;
+	sf::RenderWindow* window;
```

Comment:
Don't mean to be pedantic, but this use of asterisk is inconsistent with other examples in PR.